integration-tests/resteasy-jackson/src/main/java/io/quarkus/it/resteasy/jackson/GreetingDynamicFeature.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.it.resteasy.jackson;
+
+import javax.enterprise.inject.Instance;
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Provider
+public class GreetingDynamicFeature implements DynamicFeature {
```

Comment:
I'll prefer it if the test is moved to `integration-tests/smallrye-config`. I know that this is not smallrye-config related, but I would like to have a central piece where all (or most) of the config stuff is tested, so we don't have to go around and run multiple projects.